### PR TITLE
[Merged by Bors] - chore: add badge to open mathlib in Codespaces

### PR DESCRIPTION
**Pull request by robertylewis**
---

Requested by @AlexKontorovich at https://leanprover.zulipchat.com/#narrow/channel/113488-general/topic/adding.20codespaces.20link.20to.20main.20mathlib4.20website

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by robertylewis** [comment]
Hmm, the default badge is much bigger than the others: 
image

**Comment by AlexKontorovich** [comment]
Would it be possible to also add a big standalone button in the Installation section please, right above the same for Gitpod? Thanks!!

**Comment by robertylewis** [comment]
As far as I can tell there's no existing badge that matches the others at the top (and yes, this is an official GitHub badge).

It lines up terribly side by side with the Gitpod badge, so this might be the best we get without designing our own:
image

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ as well as tactics that use the former and allow to develop the latter.
 ## Installation
 
 You can find detailed instructions to install Lean, mathlib, and supporting tools on [our website](https://leanprover-community.github.io/get_started.html).
-Alternatively, click on the button below to open a Gitpod workspace containing the project.
+Alternatively, click on one of the buttons below to open a GitHub Codespace or a Gitpod workspace containing the project.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/leanprover-community/mathlib4)
```
Comment (eric-wieser) [inline README.md:17]: nit: might look nicer to have these two buttons on the same line, by deleting the blank line between them.

Comment (robertylewis) [reply README.md:17]: It does not. They don't line up whatsoever.

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**